src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContractSerializer.cs

```diff
@@ -32,6 +32,18 @@ public sealed class DataContractSerializer : XmlObjectSerializer
 
         private static SerializationOption s_option = IsReflectionBackupAllowed() ? SerializationOption.ReflectionAsBackup : SerializationOption.CodeGenOnly;
         private static bool s_optionAlreadySet;
+
+        internal static readonly UTF8Encoding SafeUTF8 = new UTF8Encoding(false, false);
```

Comment:
Why is this called "Safe"?

Reply:
I followed convention used in this repo

Reply:
I'll remove that Safe prefix and replace with NoBom suffix. The `Safe` term seems to be used only by private methods